app/_kong_plugins/app-dynamics/index.md

```diff
@@ -29,11 +29,121 @@ topologies:
+for more information about the configuration parameters.
+
+{:.info}
+> All non-default environment variables in the table **must** be set.
+
+The plugin uses the following environment
+variables:
+
+| Variable | Description | Type | Default |
+|--|--|--|--|
+| `KONG_APPD_CONTROLLER_HOST` | Hostname of the AppDynamics controller. | String | |
+| `KONG_APPD_CONTROLLER_PORT` | Port number to use to communicate with the controller. | Integer | `443` |
+| `KONG_APPD_CONTROLLER_ACCOUNT` | Account name to use with controller. | String | |
+| `KONG_APPD_CONTROLLER_ACCESS_KEY` | Access key to use with the AppDynamics controller. | String |
+| `KONG_APPD_LOGGING_LEVEL` | Logging level of the AppDynamics SDK agent. | Integer | `2` |
+| `KONG_APPD_LOGGING_LOG_DIR` | Directory into which agent log files are written. | String | `/tmp/appd` |
+| `KONG_APPD_TIER_NAME` | Tier name to use for business transactions. | String | |
+| `KONG_APPD_APP_NAME` | Application name to report to AppDynamics. | String | `Kong` |
+| `KONG_APPD_NODE_NAME` | Node name to report to AppDynamics. This value defaults to the system's hostname.| String | `hostname` |
+| `KONG_APPD_INIT_TIMEOUT_MS` | Maximum time to wait for a controller connection when starting, in milliseconds. | Integer | `5000` |
+| `KONG_APPD_CONTROLLER_USE_SSL` | Use SSL encryption in controller communication. `true`, `on`, or `1` are all interpreted as `True`, any other value is considered `false`.| Boolean | `on` |
+| `KONG_APPD_CONTROLLER_HTTP_PROXY_HOST` | Hostname of proxy to use to communicate with controller. | String |  |
+| `KONG_APPD_CONTROLLER_HTTP_PROXY_PORT` | Port number of controller proxy. | Integer |  |
+| `KONG_APPD_CONTROLLER_HTTP_PROXY_USERNAME` | Username to use to identify to proxy. This value is a string that is never shown in logs. This value can be specified as a vault reference.| String |  |
+| `KONG_APPD_CONTROLLER_HTTP_PROXY_PASSWORD` | Password to use to identify to proxy. This value is a string that is never shown in logs. This value can be specified as a vault reference.| String |  |
+| `KONG_APPD_CONTROLLER_CERTIFICATE_FILE` | Path to a self-signed certificate file. For example, `/etc/kong/certs/ca-certs.pem`. <br><br>_Available starting in {{site.base_gateway}} 3.4.3.3_ | String | | 
+| `KONG_APPD_CONTROLLER_CERTIFICATE_DIR` | Path to a certificate directory. For example, `/etc/kong/certs/`. <br><br> _Available starting in {{site.base_gateway}} 3.4.3.3_ | String | | 
+| `KONG_APPD_CONTROLLER_CERTIFICATE_FILE` | Path to a self-signed certificate file. For example, `/etc/kong/certs/ca-certs.pem`. <br><br>_Available starting in {{site.base_gateway}} 3.6.x_ | String | | 
+| `KONG_APPD_CONTROLLER_CERTIFICATE_DIR` | Path to a certificate directory. For example, `/etc/kong/certs/`. <br><br>_Available starting in {{site.base_gateway}} 3.6.x_ | String | | 
```

Comment:
```suggestion
| `KONG_APPD_CONTROLLER_CERTIFICATE_FILE` | Path to a self-signed certificate file. For example, `/etc/kong/certs/ca-certs.pem`. <br><br>_Available starting in {{site.base_gateway}} 3.4.3.3_ | String | | 
| `KONG_APPD_CONTROLLER_CERTIFICATE_DIR` | Path to a certificate directory. For example, `/etc/kong/certs/`. <br><br> _Available starting in {{site.base_gateway}} 3.4.3.3_ | String | | 
```
I can't see any difference between these. I'm guessing it was probably a backport that created some dupe table entries/version tags.